src/components/Authentication/Register.vue

```diff
@@ -7,8 +7,15 @@
       :key="err",
       :value="true",
       type="error") {{ err }}
+  v-alert.mt-0(
+      dismissible,
+      transition="scale-transition",
+      v-for="info in alerts.infos",
+      :key="info",
+      :value="true",
+      type="info") {{ info }}
```

Comment:
Small nitpick: can we color this as primary?  There's a slight shade difference here.

![screenshot_20190128_092335](https://user-images.githubusercontent.com/4214172/51842554-27a9c680-22df-11e9-82b3-c5ae99b05946.png)

Reply:
I think some difference could make it stand out more. Generally, I am thinking since Vuetify has built-in types for v-alert, we should use it semantically correctly because colors are customizable in sub-applications. Following Vuetify would make GWC fit in sub-application better. Does this sound reasonable?

Reply:
I think an intentional difference, like having a unique `alert:info` color, would be fine, but these shades are so close that I don't think it stands out so much as looks like a mistake.  Vuetify's behavior is to use the `info` color for info alerts, which is by default the same as the primary color.  

I suppose a more correct way to fix this would be to override `info` in `utils/vuetifyConfig.js` to be the same as our overridden primary.  Girder Web Components already makes you override that config in order to change colors and themes.  Could we do that?

Reply:
By default, Vuetify has different colors for primary and info. It a set of two colors has simlar hue. It actually looks so bad. I could change info to a color has similar hue of the customized primrary. BTW: What is the reason we customized default Vuetify's color?

Default:
![image](https://user-images.githubusercontent.com/3123478/51845879-6db65880-22e6-11e9-9572-a0b9e761a64e.png)

Changed info:
(Same Hue as primary but lighter)
![image](https://user-images.githubusercontent.com/3123478/51846221-28465b00-22e7-11e9-8d5b-5a241d53c50c.png)

Reply:
Oh yeah that does look weird.  Alright, let's not worry about the color issue here.  There may be a larger discussion on color to be had.

I'm happy to merge this as-is if that's ok.

Reply:
Cool, could you approve this PR again? (Becuase I pushed again after you approved it). Thank you for reviewing this PR. 🙂